qemu/tests/cfg/timerdevice_check_ntp_offset.cfg

```diff
@@ -0,0 +1,31 @@
+- timerdevice_check_ntp_offset:
+    only i386, x86_64
+    only Windows
+    type = timerdevice_check_ntp_offset
+    start_vm = no
+    qemu_stop = off
+    ntp_cmd = "(systemctl stop chronyd || service ntpdate stop)"
+    ntp_cmd += " && (chronyd -q 'server clock.redhat.com iburst'"
+    ntp_cmd +=  " || ntpdate clock.redhat.com)"
+    diskspd_dir = "diskspd"
+    diskspd_name = "diskspd.exe"
+    dst_path = "C:\"
+    ntp_dir = "Meinberg_NTP"
+    ntp_name = "ntp-4.2.8p14-win32-setup.exe"
+    ntp_dst_path = "C:\Meinberg_NTP"
+    ntp_unattend_file = "install_ntp.ini"
+    ntp_conf = "ntp.conf"
```

Comment:
where is the usage for `ntp_conf` ?   I guest when run `ntp-4.2.8p14-win32-setup.exe`, it will use the conf under the same path by default.

Reply:
It is used by install_ntp.ini.

Reply:
I know this file `ntp.conf` is useful, my point is: no usage for the param `ntp_conf`.

Reply:
In the `install_ntp.ini`, it use the file with a absolute path, nothing to do with the param `ntp_conf`